ilastik/applets/tracking/base/trackingBaseDataExportApplet.py

```diff
@@ -185,4 +203,79 @@ def _configure_operator_with_parsed_args(cls, parsed_args, opTrackingDataExport)
         # configure super operator
         DataExportApplet._configure_operator_with_parsed_args(parsed_args, opTrackingDataExport)
 
+    @property
+    def op(self):
+        return self.__topLevelOperator
+
+    def _export_with_plugin(self, lane_index: int, checkOverwriteFiles: bool, pluginName: str) -> bool:
+        argsSlot = self.op.AdditionalPluginArguments
+        pluginInfo = pluginManager.getPluginByName(pluginName, category="TrackingExportFormats")
+
+        if pluginInfo is None:
+            logger.error("Could not find selected plugin %s", pluginName)
+            return False
+
+        plugin = pluginInfo.plugin_object
+        logger.info("Exporting tracking result using %s", pluginName)
+
+        name_format = self.op.getLane(lane_index).OutputFilenameFormat.value
+        partially_formatted_name = self.getPartiallyFormattedName(lane_index, name_format)
+
+        if plugin.exportsToFile:
+            filename = partially_formatted_name
+            if os.path.basename(filename) == '':
+                filename = os.path.join(filename, 'pluginExport.txt')
+        else:
+            filename = partially_formatted_name
+
+        if not filename:
+            logger.error("Cannot export from plugin with empty output filename")
+            return False
+
+        self.progressSignal(-1)
+
+        status = self._doPluginExport(lane_index, filename, plugin, checkOverwriteFiles, argsSlot)
+
+        self.progressSignal(100)
+
+        if not status:
+            return False
+
+        logger.info("Export done")
+        return True
+
+    def post_process_lane_export(self, lane_index: int, checkOverwriteFiles: bool = False):
+        # `checkOverwriteFiles` parameter ensures we check only once for files that could be overwritten, pop up
+        # the MessageBox and then don't export. For the next round we click the export button,
+        # we really want it to export, so checkOverwriteFiles=False.
+        # Plugin export if selected
+        op = self.topLevelOperator
+
+        logger.info("Export source is: %s", op.SelectedExportSource.value)
+        plugin_export_selected = op.SelectedExportSource.value == OpTrackingBaseDataExport.PluginOnlyName
+
+        if plugin_export_selected:
+            logger.info("Export source plugin selected!")
+            return self._export_with_plugin(lane_index, checkOverwriteFiles, op.SelectedPlugin.value)
+
+        return True
+
+    def getPartiallyFormattedName(self, lane_index: int, path_format_string: str) -> str:
```

Comment:
this doesn't use `lane_index`, any reason to keep it?

Reply:
Fixed use lane index to fetch RawDatasetInfo. Not sure if it has any impact because manual tracking limits you to 1 dataset.